examples/agents_visualizations.jl

```diff
@@ -228,7 +230,8 @@ fig
 # and plot it with [`abmplot`](@ref).
 using Graphs
 using ColorTypes
-sir_model = Models.sir()
+using .AgentsTestModels: sir #hide
+sir_model = sir()
```

Comment:
Actually I'm a bit concerned about this: does it mean that the user won't be able to copy-paste the code in the example? It will error right? Can we instead use a sort of relative path from the Agents module to import it so that it should also work for a user?

Reply:
Also `include("../test/AgentsTestModels/AgentsTestModels.jl") #hide` has the same problem, I think that adding `#hide` makes the problem worse

Reply:
or maybe another solution would be to put those models used in the documentation inside `AgentsExampleZoo` and then import them from there?

Reply:
mmmh, given the structure of `AgentsExampleZoo` which just uses docs, to me it seems better the other way

Reply:
ok, It should be `include(joinpath(splitpath(pathof(Agents))[1:end-2]..., "test/AgentsTestModels/AgentsTestModels.jl"))`, maybe we can call it something like `include_Agents_test_models()`

Reply:
> does it mean that the user won't be able to copy-paste the code in the example? 

Not with a simple click but if they just go the the link mentioned in the docs themselves, they can copy the model quite easily.

> maybe we can call it this way Agents.include_test_models()

Creating such a helper function is just a more complicated way to have the `Models` submodule in the package itself. I'm against such a half-baked solution. Either we don't expose those predefined models to the users at all or we put all of this in `AgentsExampleZoo`. I've actually thought about the latter quite a bit while doing this PR and I would be in favour of that. Give me your opinion and we'll decide what to do. :)